TSC-Charter.md

```diff
@@ -74,9 +76,43 @@ including:
 * Development process and any coding standards.
 * Mediating technical conflicts between Collaborators or Foundation
   projects.
+* Establishing guidelines for communication by project contributors about the
+  technical development processes of the project, releases and release announcements,
+  technical documentation, contributor onboarding and conduct, project governance,
+  and technical direction.
 
 The TSC will define Node.js project’s release vehicles.
 
+### Exclusions and limitations of TSC responsibilities
```

Comment:
I really think this should be part of the CPC set of documents for _all_ projects.

Reply:
I agree

Reply:
Yes. I think this change would need to go in https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-6-non-responsibilities-of-the-cpc.